src/cerbos-compile-and-test.ts

```diff
@@ -29,32 +28,35 @@ async function cerbosCompileAndTest(
   }
 
   core.info(`Command to run: ${command}`)
-
   core.startGroup('cerbos compile results')
-  let stdout = ''
-  try {
-    stdout = child.execSync(command, {
-      encoding: 'utf8'
-    })
-  } catch (error) {
-    const execSyncError = asExecSyncException(error)
-
-    switch (execSyncError.status) {
-      case 1: // returns 1 if there are compilation errors
-        core.setFailed(`Compilation errors detected: ${error}`)
-        break
-      default:
-        core.setFailed(`Failed to launch Cerbos: ${error}`)
-        break
+  child.exec(command, (err, stdout, stderr) => {
+    if (err || stderr) {
+      core.setFailed(`Error(s) occurred`)
+      if (err) {
```

Comment:
Cerbos exits with code 3 if policies have issues and code 4 if the tests fail. Can we handle those cases here and print out the relevant error messages like "Compilation failed" or "Tests failed"?

https://github.com/cerbos/cerbos/blob/1f6ba2c0cead154b1e663da8116d5a98ce58965e/cmd/cerbos/main.go#L45-L55